.idea/codeStyles/Project.xml

```diff
@@ -17,20 +17,6 @@
     <editorconfig>
       <option name="ENABLED" value="false" />
     </editorconfig>
-    <codeStyleSettings language="CSS">
-      <arrangement>
-        <rules>
-          <section>
-            <rule>
-              <match>
-                <NAME>.*</NAME>
-              </match>
-              <order>BY_NAME</order>
-            </rule>
-          </section>
-        </rules>
-      </arrangement>
-    </codeStyleSettings>
```

Comment:
You don't wanna sort the CSS styles by name anymore?

Reply:
Uhm... Actually I want that. O_o